content/docs/faq.md

```diff
@@ -31,8 +31,10 @@ If you encounter a problem while using packit or Packit Service, please open an
 In case of any other questions, feel free to contact
 [us](https://github.com/orgs/packit/teams/the-packit-team):
 
-- **#packit** on [Libera.Chat](https://libera.chat/)
+- [**#packit:fedora.im**](https://chat.fedoraproject.org/#/room/#packit:fedora.im) (Element/Matrix)
+- [**#packit:libera.chat**](https://libera.chat) (IRC)
 - **hello@packit.dev**
+- [**@packit@fosstodon.org**](https://fosstodon.org/@packit) (Mastodon)
```

Comment:
Do we want to maintain the contacts in two places?

What about linking the one on the main page from here?

Reply:
I agree with not having it in two places, but I did it the other way round, i.e. left the contacts in FAQ and linked them from the main page. The main page is the entry point and I don't think the info about how to contact us is something someone who reads about Packit for the first time needs to know to decide whether Packit is a good fit for them.